lib/msal-node/test/client/ClientCredentialClient.spec.ts

```diff
@@ -437,19 +443,55 @@ describe("ClientCredentialClient unit tests", () => {
                             .filter((key: string) => key.includes("claims="))[0]
                             .split("claims=")[1]
                     )
-                ).toEqual(mergedClaims);
+                ).toEqual(CAE_CONSTANTS.MERGED_EMPTY_CLAIMS);
 
+                // acquire a token (without changing anything) and verify that it comes from the cache
                 const cachedAuthResult = (await client.acquireToken(
                     clientCredentialRequest
                 )) as AuthenticationResult;
                 expect(cachedAuthResult.accessToken).toEqual(
                     CONFIDENTIAL_CLIENT_AUTHENTICATION_RESULT.body.access_token
                 );
                 expect(cachedAuthResult.fromCache).toBe(true);
+
+                // acquire a token with a client that has client capabilities, and has claims in the request
+                clientCredentialRequest.claims = claims;
+                const authResult2 = (await client.acquireToken(
+                    clientCredentialRequest
+                )) as AuthenticationResult;
+                expect(authResult2.accessToken).toEqual(
+                    CONFIDENTIAL_CLIENT_AUTHENTICATION_RESULT.body.access_token
+                );
+                expect(authResult2.fromCache).toBe(false);
```

Comment:
Can you please add 2 more requests to this test: 

- another request with no claims - expect the result to come from cache
- another request with claims - expect the result to come from IDP

in both cases client capability continues to be set.

Reply:
I don't understand, I have these tests already. You want me to repeat them?